astropy/visualization/wcsaxes/settings.py

```diff
@@ -0,0 +1,7 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import print_function, division, absolute_import
+
+COORDINATE_RANGE_SAMPLES = 50
```

Comment:
Should the config system be used for these? Or are they never meant to be changed?